README.md

```diff
@@ -132,85 +132,86 @@ To add your own ICO you would have to:
 
 1. Fork the project - Github won't allow you to create pull request otherwise.
 
-2. Include the ICO Smart-Contract [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) in the [Smart_Contracts](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/smart_contracts) folder
-    this ABI is generated from the smart-contract source code and can be found in some cases in [etherscan](https://etherscan.io/)
+2. Include the ICO Smart-Contract [ABI](https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI) in the [Smart_Contracts](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/smart_contracts) folder, name the files with the contract address,
+    this ABI is generated from the smart-contract source code and can be found in some cases in [etherscan](https://etherscan.io/), sometimes you have to add two ABIs files, crowd-sale ABI and token contract ABI. 
 
-3. Configure [config.js](https://github.com/Neufund/ico-transparency-monitor/blob/master/src/config.js)
-  and add the required information from the smart contract manually. This will require some JavaScript modifications.
+3. Add new file with ICO configuration with required information from the smart contract manually into [icos_config](https://github.com/Neufund/ico-transparency-monitor/tree/master/src/icos_config) directory and it will be add automatically, This will require some JavaScript modifications.
```

Comment:
Add a new file with ICO configuration with required information from the smart contract manually into
`create a new file for your ICO configurations and add it to [ICO CONFIG] directory this file should include ... and will be added automatically. This will require some JavaScript modifications`